Makefile

```diff
@@ -94,39 +90,24 @@ tag: ## Report the git tag that would be used for the images
 version: ## Report the version that would be put in the binary
 	@echo $(VERSION)
 
-
-fmt-check: ## Check all source code formatting
-	@if [ -n "$(shell $(BUILD_CMD) gofmt -l ${GO_FILES})" ]; then \
-	  $(BUILD_CMD) gofmt -s -e -d ${GO_FILES}; \
-	  exit 1; \
-	fi
-
-fmt:   ## Format all source code files
-	$(BUILD_CMD) gofmt -w -s ${GO_FILES}
+fmt: golangci-lint  ## Format all source code files
+	@$(BUILD_CMD) $(LINTER) run --fix ./
```

Comment:
standardize on golangci-lint, this also attempts to auto fix any issues similar to the old fmt target.